includes/Checker/Abstract_Check_Runner.php

```diff
@@ -186,14 +205,40 @@ final public function set_experimental_flag( $include_experimental ) {
 		if ( $this->initialized_early ) {
 			if ( $include_experimental !== $this->get_include_experimental_param() ) {
 				throw new Exception(
-					__( 'Invalid flag: The include-experimental flag does not match the original request parameter.', 'plugin-check' )
+					sprintf(
+						/* translators: %s: include-experimental */
+						__( 'Invalid flag: The %s flag does not match the original request parameter.', 'plugin-check' ),
```

Comment:
Good idea to consolidate the strings, just one rewording suggestion:
```suggestion
						__( 'Invalid flag: The %s value does not match the original request parameter.', 'plugin-check' ),
```